src/ServiceManagement/RecoveryServices/Commands.RecoveryServices/PSRecoveryServicesClient/PSRecoveryServicesClient.cs

```diff
@@ -259,6 +261,44 @@ private SiteRecoveryManagementClient GetSiteRecoveryClient()
 
             return siteRecoveryClient;
         }
+
+        /// <summary>
+        /// Gets Vault Upgrade client.
+        /// </summary>
+        /// <param name="location">Resource location.</param>
+        /// <param name="resourceName">Resource Name.</param>
+        /// <param name="resourceType">Resource Type.</param>
+        /// <returns>Vault Upgrade Management client.</returns>
+        private RecoveryServicesVaultUpgradeManagementClient GetVaultUpgradeClient(
+            string location,
+            string resourceName,
+            string resourceType)
+        {
+            string resourceNamespace = Environment.GetEnvironmentVariable("RESOURCE_NAMESPACE");
```

Comment:
probably add comments around this why this is used/ present in code. also suggest renaming the variable to "RecoveryServices_VaultUpgrade_ResourceNamespace" to be very explicit